backend/tests/test_llm_providers_api.py

```
@@ -0,0 +1,108 @@
+    def __init__(self):
+        self.items = []
+        self.audits = []
+
+    async def execute(self, stmt):
+        class MockResult:
+            def scalars(self):
+                class MockScalars:
+                    def all(self):
+                        return self.items
+                    def first(self):
+                        if self.items: return self.items[0]
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)